examples/kubernetes/1.10/cilium-crio-ds.yaml

```diff
@@ -50,7 +50,7 @@ spec:
                 optional: true
                 key: clean-cilium-state
       containers:
-      - image: docker.io/cilium/cilium:v1.1.0-rc4
+      - image: docker.io/cilium/cilium:v1.1.0
         imagePullPolicy: Always
```

Comment:
`imagePullPolicy: IfNotPresent`, change it in the `.sed` file for the 1.1 branch and regenerate again